.mise/tasks/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#MISE description="Build the project"
+
+#USAGE flag "-s --spm" help="Build through SPM"
+#USAGE flag "-v --linux-vm" help="Build virtualizing Linux"
+
+if [ "$usage_spm" = "true" ]; then
+    if [ "$usage_linux_vm" = "true" ]; then
+        if command -v podman &> /dev/null; then
```

Comment:
Is `--linux-vm` valid without `--smp`? If not, maybe I would default to spm if linux-vm is passed

Reply:
I removed `--spm` making it the default.